pkg/dryrun/dryrun_utils.go

```diff
@@ -0,0 +1,273 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package dryrun
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"sort"
+
+	"github.com/spf13/cobra"
+
+	policyv1 "open-cluster-management.io/config-policy-controller/api/v1"
+)
+
+var ResetColor = "\033[0m"
```

Comment:
nitpick (if this works)
```suggestion
const resetColor = "\033[0m"
```